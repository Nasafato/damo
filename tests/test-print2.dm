def main() : int
{
    print_string("Hello, literal world!");
    print_string("And here's another one");
    return 0;
}
