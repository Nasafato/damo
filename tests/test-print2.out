Hello, literal world!
And here's another one
